lib/App/Rakubrew.pm

```diff
@@ -311,11 +311,13 @@ EOL
 
     } elsif ($arg eq 'build-zef') {
         my $version = get_version();
+        my $zef_version = shift(@args);
         if (!$version) {
-            say STDERR "$brew_name: No version set.";
-            exit 1;
+          say STDERR "$brew_name: No version set.";
+          exit 1;
         }
-        App::Rakubrew::Build::build_zef($version);
+	say "Building zef ", $zef_version?$zef_version:"latest";
```

Comment:
There is still a tab here. Also 2-space indentation above.

Reply:
Sorry...